pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -3,13 +3,13 @@ permalink: /documentation/getting-started/developers/phase-two-compile/
 layout: styleguide
 title: "Phase 2: Compile"
 category: How to use USWDS
-lead: Compile design system source code into browser-readable files
+lead: Compile Design System source code into browser-readable files.
 type: docs
 subnav:
   - text: "Step 1: Set up your project’s Sass entry point"
     href: "#step-1-set-up-your-projects-sass-entry-point"
-  - text: "Step 2: Install uswds-gulp"
-    href: "#step-2-install-uswds-gulp"
+  - text: "Step 2: Install uswds-compile"
+    href: "#step-2-install-uswds-compile"
   - text: "Step 3: Add configuration files"
```

Comment:
@mikki-stacey I noticed the step name in the side navigation does not match the step name in the content - Do we want the nav to say "Step 3: Create a gulpfile" and "Step 4: Create path settings and export compile functions" to match?

Reply:
@amyleadem , great catch! Yes, please revise the side nav to match.